pay-queue/tests/integration/test_payment_reconciliations.py

```diff
@@ -686,3 +686,59 @@ def mock_cms(cfs_account: CfsAccountModel,
     assert pay_account.credit == onac_amount + cm_amount - cm_used_amount
     credit = CreditModel.find_by_id(credit_id)
     assert credit.remaining_amount == cm_amount - cm_used_amount
+
+
+def test_unconsolidated_invoices_errors(session, app, client, mocker):
+    """Test error scenarios for unconsolidated invoices in the reconciliation worker."""
+    # Create payment account
+    cfs_account_number = '1234'
+    pay_account: PaymentAccountModel = factory_create_online_banking_account(status=CfsAccountStatus.ACTIVE.value,
+                                                                             cfs_account=cfs_account_number)
+
+    invoice: InvoiceModel = factory_invoice(payment_account=pay_account, total=100, service_fees=10.0,
+                                            payment_method_code=PaymentMethod.ONLINE_BANKING.value)
+    factory_payment_line_item(invoice_id=invoice.id, filing_fees=90.0,
+                              service_fees=10.0, total=90.0)
+    invoice_number = '1234567890'
+    factory_invoice_reference(invoice_id=invoice.id, invoice_number=invoice_number)
+    factory_invoice_reference(invoice_id=invoice.id, invoice_number=invoice_number)
+    invoice.invoice_status_code = InvoiceStatus.SETTLEMENT_SCHEDULED.value
+    invoice = invoice.save()
+    invoice_id = invoice.id
+    total = invoice.total
+
+    error_messages = [{'error': 'Test error message', 'row': 'row 2'}]
+    mocker.patch('pay_queue.services.payment_reconciliations._process_file_content',
+                 return_value=(True, error_messages))
+    mock_send_error_email = mocker.patch('pay_queue.services.payment_reconciliations._send_error_email')
+
+    # Create a settlement file and publish with multiple active references
```

Comment:
I think this is good, because I can't see multiple active references in the code below

Reply:
it's not, thanks point out, I removed